applications/minotari_node/src/bootstrap.rs

```diff
@@ -125,6 +128,11 @@ where B: BlockchainBackend + 'static
                 base_node_config.network,
                 self.node_identity.clone(),
                 publisher,
+                FixedHash::from_hex(get_static_genesis_block_hash(
```

Comment:
Perhaps we should add a prefix to ensure the prologue is fully qualified

```rust
    fn block_hash_to_noise_prologue(block_hash: &FixedHash) -> Vec<u8> {
        const PREFIX: &[u8] = b"com.tari.comms.noise.";
        let mut buf = vec![0u8; PREFIX.len() + FixedHash::byte_size()];
        buf[..PREFIX.len()].copy_from_slice(PREFIX);
        buf[PREFIX.len()..].copy_from_slice(&**block_hash);
        buf
    }

Reply:
Good idea